index.bs

```diff
@@ -467,6 +468,11 @@ To <dfn>initialize WebTransport over HTTP</dfn> for a given |transport| and
 
 ## Methods ##  {#webtransport-methods}
 
+: <dfn for="WebTransport" method>setDatagramWritableQueueExpirationDuration()</dfn>
+:: Sets the expiration of the {{datagramWritable}} queue, in milliseconds.
+   {{writer}}.{{ready}} is not fulfilled when there are more outgoing datagrams
+   neither sent nor discarded due to a timeout.
```

Comment:
The `ready` API is defined in algorithms in the streams spec, so we need to influence inputs to those algorithms. https://github.com/w3c/webtransport/issues/185